global.d.ts

```diff
@@ -0,0 +1,7 @@
+declare global {
+  interface Window {
+    filterBySection: (section: any) => void;
+  }
+}
+
+export {};
```

Comment:
Did this to get through some ts errors on global `Window` bindings. 

However... I'll be honest, I've never worked with Typescript in a "pure JS" context like this one. TS came along strongly after I had already been pretty entrenched in React, where we don't manage widnow bindings in such a low-level way. I'll need to do some research on this one.